pkg/config/install_requirements_test.go

```diff
@@ -356,3 +357,53 @@ func TestHelmfileRequirementValues(t *testing.T) {
 	assert.FileExists(t, file)
 	assert.FileExists(t, valuesFile, "file %s should exist", valuesFile)
 }
+
+func Test_LoadRequirementsConfig(t *testing.T) {
+	t.Parallel()
+
+	var gitPublicTests = []struct {
+		requirementsPath   string
+		createRequirements bool
+		loadSuccessful     bool
```

Comment:
This is an unused filed. Or am I missing something?

Reply:
Nope, not at all. I had initially something else in mind, but in the end, never needed this flag. Going to remove it.